dashboard/models.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
don't know if this was your decision initially, but why are these models being defined in the 'dashboard' app? it makes logical sense for Profile/Education/etc to be defined in the 'profiles' app. Same for Course/CourseRun/Program being defined in the 'courses' app. i don't understand why these are defined in 'dashboard' when it seems like 'courses' would make a lot more sense